ui/app/src/views/ViewDashboard.tsx

```diff
@@ -48,7 +48,7 @@ function ViewDashboard() {
       }}
     >
       <ErrorBoundary FallbackComponent={ErrorAlert}>
-        <PluginRegistry pluginLoader={bundledPluginLoader}>
+        <PluginRegistry pluginLoader={bundledPluginLoader} defaultPanelKind="TimeSeriesChart">
```

Comment:
Nice! I think in the future, we may want to expand this to be more generic (e.g. `defaultPluginKinds` or something) since I think other places where you need to select a plugin have a similar problem. (Other places would be variable kind selection when editing a variable and query kind selection when editing a query in a panel.)

Reply:
Good call. Let me take a quick timeboxed look at if that's an easy refactor now since that could avoid a breaking change in the future.

Reply:
That was an easy refactor! Tagging you to give this one more pass before merging since it was a significant code change.